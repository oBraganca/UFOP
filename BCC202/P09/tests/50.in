50
Esther 746369 6.5
Amanda 776373 9.0
Raul 981102 2.0
MariaLuiza 727160 1.3
Felipe 250545 7.2
Gustavo 55006 8.8
Manuela 699451 2.4
AnaCecilia 991836 2.2
Isaac 686690 5.3
Livia 653889 8.6
Henrique 997226 3.0
Daniel 5393 8.9
Joaquim 575259 9.5
Mariana 368519 9.4
Vicente 168524 9.2
Davi 964937 5.4
Emanuel 219431 6.4
Augusto 910271 2.7
Helena 166431 1.5
MariaAlice 452161 4.0
Enzo 410007 9.5
Matheus 377334 7.5
LuizHenrique 105841 2.3
Samuel 365571 5.0
DaviMiguel 520881 5.2
Beatriz 932258 2.9
Allana 703897 1.6
Bento 214830 4.2
LuizMiguel 186576 9.1
Rodrigo 510209 7.9
AnaLuiza 798850 5.4
Anthony 571013 0.6
Emanuelly 791315 9.9
Vinicius 713076 2.0
Carolina 325398 1.2
Gael 153107 8.5
Miguel 809636 7.5
MariaHeloisa 975675 5.0
Antonio 502439 4.4
Kaique 65048 1.0
EnzoMiguel 855826 5.7
Benjamin 556235 1.7
Isabel 218877 5.3
JoaoVictor 289539 7.2
Eduarda 992883 3.0
MariaJulia 243067 3.8
Anthony 201694 6.6
Lavinia 749618 1.7
Martin 57157 9.1
Mirella 350796 5.7
