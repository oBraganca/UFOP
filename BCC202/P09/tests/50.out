Anthony 571013 0.6
Kaique 65048 1.0
Carolina 325398 1.2
MariaLuiza 727160 1.3
Helena 166431 1.5
Allana 703897 1.6
Benjamin 556235 1.7
Lavinia 749618 1.7
Vinicius 713076 2.0
Raul 981102 2.0
AnaCecilia 991836 2.2
LuizHenrique 105841 2.3
Manuela 699451 2.4
Augusto 910271 2.7
Beatriz 932258 2.9
Eduarda 992883 3.0
Henrique 997226 3.0
MariaJulia 243067 3.8
MariaAlice 452161 4.0
Bento 214830 4.2
Antonio 502439 4.4
Samuel 365571 5.0
MariaHeloisa 975675 5.0
DaviMiguel 520881 5.2
Isabel 218877 5.3
Isaac 686690 5.3
AnaLuiza 798850 5.4
Davi 964937 5.4
Mirella 350796 5.7
EnzoMiguel 855826 5.7
Emanuel 219431 6.4
Esther 746369 6.5
Anthony 201694 6.6
Felipe 250545 7.2
JoaoVictor 289539 7.2
Matheus 377334 7.5
Miguel 809636 7.5
Rodrigo 510209 7.9
Gael 153107 8.5
Livia 653889 8.6
Gustavo 55006 8.8
Daniel 5393 8.9
Amanda 776373 9.0
Martin 57157 9.1
LuizMiguel 186576 9.1
Vicente 168524 9.2
Mariana 368519 9.4
Enzo 410007 9.5
Joaquim 575259 9.5
Emanuelly 791315 9.9
